client/signup/steps/domains/index.jsx

```diff
@@ -128,7 +128,8 @@ class DomainsStep extends React.Component {
 			! props.isPlanStepFulfilled &&
 			'variantShowUpdates' === abtest( 'domainStepCopyUpdates' )
 		) {
-			this.showTestCopy = true;
+			// this.showTestCopy = true;
+			this.showDesignUpdate = true;
```

Comment:
To clarify, this is a temporary update that will be removed once https://github.com/Automattic/wp-calypso/pull/39276 is merged. This PR will not merge before https://github.com/Automattic/wp-calypso/pull/39276.